src/components/ArticleController.jsx

```diff
@@ -4,14 +4,19 @@ import FalcorController from 'lib/falcor/FalcorController';
 
 export default class ArticleController extends FalcorController {
   static getFalcorPath(params) {
-    // Format: thegazelle.org/issue/:issueId/:category/:articleSlug
-    return ["data", "articles", params.issueId, params.category, params.articleSlug, ["title", "slug", "html"]];
+    // Format: thegazelle.org/issue/:issueId/:articleCategory/:articleSlug
+    return ["issues", params.issueId, "articles", params.articleCategory, params.articleSlug, ["title", "html"]];
```

Comment:
How I have posts.js set up, the path needs to be this way. In posts.js, it's set up as
```
issues: {
  "1": {
    articles: {
      "category": {
        "articleSlug": { $type: "ref", value: ["articlesById", 1] }
      }
    }
  }
}
```
I changed it from how I originally had it set up as just a list of articles so that it would be more managable at scale. 

With the falcorPath

`return ["articles", "params.articleSlug", ["title", "html"]]`

I get an error that the data is not in the cache because it's not organized like that.